package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ambrosus-node-contracts",
   "author": "Ambrosus",
   "description": "Smart contracts used in AMB-NET",
-  "version": "0.0.26",
+  "version": "0.0.27",
```

Comment:
Please update this one more time when you rebase on master (0.0.27 is already taken)